packages/dataviews/CHANGELOG.automattic.md

```diff
@@ -3,3 +3,7 @@
 ## 0.1.0
 
 - First release of the `@automattic/dataviews` package. It bundles all changes from `@wordpress/dataviews` 4.18.0.
+
```

Comment:
We should order the Changelog from newest to oldest, so it's easier to skim and follows the common conventions for changelogs. The common use cases are 1) look at what are the latest changes or 2) add new changes. Both benefit from having a newest-to-oldest ordering.